tests/structured_conf/data/dataclasses.py

```diff
@@ -403,10 +403,10 @@ class DictExamples:
         }
     )
     int_keys: Dict[int, str] = field(default_factory=lambda: {1: "one", 2: "two"})
-
-
-@dataclass
-class DictWithEnumKeys:
+    float_keys: Dict[float, str] = field(
```

Comment:
DictExamples becomes very large.
Maybe break it into `DictValueTypes` and `DictKeyTypes`?